odl/__init__.py

```diff
@@ -16,18 +16,16 @@
 # along with ODL.  If not, see <http://www.gnu.org/licenses/>.
 
 
-"""ODL is a functional analysis library with a focus on discretization.
+"""ODL (Operator Discretization Library)
 
-ODL suppors abstract sets, linear vector spaces defined on such
-and Operators/Functionals defined on these sets. It is intended
-to be used to write general code and faciliate code reuse.
+ODL is a Python library for fast prototyping focusing on (but not
+restricted to) inverse problems.
 """
 
 from __future__ import absolute_import
 
 __version__ = '0.5.4.dev0'
-__all__ = ('diagnostics', 'discr', 'operator', 'set', 'space', 'solvers',
-           'tomo', 'trafos', 'util', 'phantom', 'deform', 'ufunc_ops')
+__all__ = ()
```

Comment:
As I understand it, the imports further down put all the subpackages in the module dictionary, so that's not the purpose of adding them to `__all__` anyway. The `__all__` here is only effective if someone does `from odl import *`, and then I guess we don't want the subpackages *themselves* to be included, only their `__all__` contents. Or do we?